### PR TITLE
add orbit start command

### PR DESCRIPTION
**Pull request by Jason-Wanxt**

### CODE REVIEW

File: arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx [hunk 1]
```diff
@@ -43,7 +43,11 @@ function sayHi() external view returns(string memory);
 
 Next, build Nitro by following steps 3-7 of the instructions in [How to build Nitro locally](/node-running/how-tos/build-nitro-locally). Note that if you've already built the Docker image, you still need run the last step to rebuild.
 
-Run Nitro by following the instructions in [How to run a full node](/node-running/how-tos/running-a-full-node#putting-it-all-together).
+Run Nitro by following the instructions (Note the instructions in [How to run a full node](/node-running/how-tos/running-a-full-node#putting-it-all-together) cann't start orbit node):
```
Comment (mahsamoosavi) [inline arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx:46]:
```suggestion
Run Nitro by following the instructions (note the instructions provided in [How to run a full node](/node-running/how-tos/running-a-full-node#putting-it-all-together) cannot be used to start an Orbit node):
```

Comment (TucksonDev) [inline arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx:46]:
```suggestion
Run Nitro with the following command:
```

Proposed change. It feels weird to link to a page when we're saying that you can't use the information in that page. Maybe we could add a link below the command with something like "for additional, optional parameters see [How to run a full node](/node-running/how-tos/running-a-full-node#optional-parameters)".
What do you think?

**Review by mahsamoosavi — APPROVED** [review]
LGTM!

**Review by symbolpunk — APPROVED** [review]
Thanks so much @Jason-Wanxt & co 🔥

File: arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx [hunk 2]
```diff
@@ -43,7 +43,17 @@ function sayHi() external view returns(string memory);
...
-Run Nitro by following the instructions in [How to run a full node](/node-running/how-tos/running-a-full-node#putting-it-all-together).
+Run Nitro with the following command:
+
+```shell
+docker run --rm -it  -v /some/local/dir/arbitrum:/home/user/.arbitrum -p 0.0.0.0:8547:8547 -p 0.0.0.0:8548:8548 @latestOrbitNodeImage@ --parent-chain.connection.url=<YourParentChainUrl> --chain.id=<YourOrbitChainId> --http.api=net,web3,eth,debug --http.corsdomain=* --http.addr=0.0.0.0 --http.vhosts=*
+```
+
+:::info
+
+Note that the instructions provided in [How to run a full node](/node-running/how-tos/running-a-full-node) **will not** work with your Orbit node, but the flags in [Command-line options](/node-running/command-line-options) **will**.
```
Comment (TucksonDev) [inline arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx:54]: Ohhh, it looks great!
One thing to note is that some of the command line options will not work with the Orbit node. We could replace them with these [optional parameters](https://nitro-docs-git-add-orbit-command-offchain-labs.vercel.app/node-running/how-tos/running-a-full-node#optional-parameters) for example. They will work and they might be useful for users 💪

Comment (symbolpunk) [reply arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx:54]: Ty @TucksonDev - https://github.com/OffchainLabs/arbitrum-docs/pull/620 is an attempt to address this

**Pull request merged**